### PR TITLE
update nextclade datasets

### PR DESCRIPTION
**Pull request by Joon-Klaps**
Add nextclade test datasets to be compatible with v3+ of nextclade

### CODE REVIEW

**Review by jfy133 — CHANGES_REQUESTED** [review]
This needs README update, particularly if tehre are different versions

**Comment by Joon-Klaps** [comment]
Please forgive my editor config reformatting the md file.

File: README.md [hunk 1]
```diff
@@ -49,56 +49,58 @@ kraken2-build --db kraken2_hs22 --build
...
-* `GCF_014621545.1_ASM1462154v1_genomic.<DOWNLOAD_DATE>.gff.gz`: Monkeypox genome GFF3 annotation file downloaded directly via [NCBI FTP](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/014/621/545/GCF_014621545.1_ASM1462154v1/GCF_014621545.1_ASM1462154v1_genomic.gff.gz)
+-   `GCF_014621545.1_ASM1462154v1_genomic.<DOWNLOAD_DATE>.fna.gz`: Monkeypox genome fasta file downloaded directly via [NCBI FTP](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/014/621/545/GCF_014621545.1_ASM1462154v1/GCF_014621545.1_ASM1462154v1_genomic.fna.gz)
+-   `GCF_014621545.1_ASM1462154v1_genomic.<DOWNLOAD_DATE>.gff.gz`: Monkeypox genome GFF3 annotation file downloaded directly via [NCBI FTP](https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/014/621/545/GCF_014621545.1_ASM1462154v1/GCF_014621545.1_ASM1462154v1_genomic.gff.gz)
+-   `nextclade_hMPXV_NC_063383.1_2024-08-27--21-28-04Z.tar.gz`: A set of input data files required for Nextclade to run an analysis on MPOX. Previous format did not require `pathogen.json` file but from v3+ it is required. File was created with `nextclade dataset get -n MPXV --tag 2024-08-27--21-28-04Z`.
```
Comment (jfy133) [inline README.md:69]: Is the previous version lited somewhere :thinking: ?

**Review by jfy133 — APPROVED** [review]

**Comment by Joon-Klaps** [comment]
Linked to https://github.com/nf-core/viralrecon/pull/446

**Comment by robsyme** [comment]
The current version of viralrecon [uses nextclade v2.12.0](https://github.com/nf-core/viralrecon/blob/3731dd3a32a67a2648ea22c2bd980c224abdaee2/modules/nf-core/nextclade/run/main.nf#L5-L8), which is incompatible with these v3 style references. If we update these configuration, won't this break the current viralrecon release?

**Pull request merged**